datamonster_api/lib/client.py

```diff
@@ -1,11 +1,15 @@
 import datetime
 import hashlib
 import hmac
+import logging
 import requests
 
 from .errors import DataMonsterError
 
 
+logging.basicConfig(level=logging.DEBUG)  # FOR TESTING PURPOSES
```

Comment:
DO NOT MERGE THIS INTO MASTER

Reply:
are we waiting for something else before this can get in?

Reply:
evan is testing

Reply:
classic evan